types/module/module.go

```diff
@@ -208,6 +212,9 @@ func (gam GenesisOnlyAppModule) LegacyQuerierHandler(*codec.LegacyAmino) sdk.Que
 // RegisterServices registers all services.
 func (gam GenesisOnlyAppModule) RegisterServices(Configurator) {}
 
+// ConsensusVersion tracks state-breaking versions of the module.
+func (gam GenesisOnlyAppModule) ConsensusVersion() uint64 { return 0 }
```

Comment:
### Proposal

Let's start the version numbers from `0`. This way, we will be able to detect wrong / empty versions. In a migrations map, a default version is 0, so we will be able to see that the migration shouldn't happen there.

Reply:
I think it's a good idea, it forces users to be explicit about versions in the MigrationMap.

Reply:
Yes, I meant `1` of course.